cluster/manifests/external-dns/deployment.yaml

```diff
@@ -35,14 +35,18 @@ spec:
         - --txt-owner-id={{ .Region }}:{{ .LocalID }}
         resources:
           limits:
-            cpu: 50m
+            cpu: 25m
```

Comment:
This isn't intended, got changed in the meantime: https://github.com/zalando-incubator/kubernetes-on-aws/commit/a65eb7611610ff1ec344e48e0963a0786b9cdc70#diff-51acfae938d9fe0cc68f5a4ca83a59ea